File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.cs
public class Solution {
    public int LongestConsecutive(int[] nums) 
    {
        HashSet<int> visited = new HashSet<int>();
        int maxconsecutive = 0;
        for(int i = 0; i < nums.Length; i++)
        {
            visited.Add(nums[i]);
        }
        foreach(int num in visited)
        {
            if(!visited.Contains(num - 1))
            {
                int newnum = num;
                int currentconsecutive = 1;
                while(visited.Contains(newnum + 1))
                {
                    currentconsecutive++;
                    newnum++;
                }
                if(currentconsecutive > maxconsecutive)
                {
                    maxconsecutive = currentconsecutive;
                }
            }
        }
        return maxconsecutive;
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-1.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) 
    {
        int maxConsecutive = 0;
        int consecutive = 0;
        for(int num : nums)
        {
            consecutive = num == 1 ? ++consecutive : 0;
            maxConsecutive = Math.max(consecutive, maxConsecutive);
        }
        return maxConsecutive;
    }
}